openweather-based-bidding/classes/sheets.gs

```diff
@@ -52,10 +64,69 @@ class SheetsApi {
    * Fetches data from sheet
    *
    * @param {string} range A1-Range
+   * @param {string} renderMode Render mode, [more info](https://developers.google.com/sheets/api/reference/rest/v4/ValueRenderOption)
    *
    * @return {!Array<!Array<!Object>>}
    */
-  get(range) {
-    return Sheets_v4.Spreadsheets.Values.get(this.spreadsheetId, range);
+  get(range, renderMode) {
+    return Sheets_v4.Spreadsheets.Values.get(
+        this.spreadsheetId,
+        range,
+        renderMode || this.defaultMode
+      )['values'];
+  }
+
+  /**
+   * Fetches data from one cell from the sheet
+   *
+   * @param {string} row Row number
+   * @param {string} col Column number
+   *
+   * @return {Object}
+   */
+  getOne(row, col) {
```

Comment:
Maybe call it `getCell()` to be more clear? Or even `getCellValue()`?

Reply:
👍🏿